packages/shared-lib/api/storage.ts

```
@@ -0,0 +1,38 @@
+    const res = uploadData({
+      path: uploadPath,
+      data: image,
+    });
+
+    const uploadedFilePath = (await res.result).path;
+
+    if (!uploadedFilePath) throw new Error("No path to file found");
+
+    return uploadedFilePath;
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)